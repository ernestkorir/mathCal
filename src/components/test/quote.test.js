import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../../../../math-magician/src/components/Quote';

it('renders correctly', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
