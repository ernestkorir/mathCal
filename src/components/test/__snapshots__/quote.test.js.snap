// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section>
  <div
    id="quote-box"
  />
</section>
`;
